src/data/set/finite.lean

```diff
@@ -236,6 +236,14 @@ fintype.of_finset (s.to_finset ∪ t.to_finset) $ by simp
 theorem finite.union {s t : set α} : finite s → finite t → finite (s ∪ t)
 | ⟨hs⟩ ⟨ht⟩ := ⟨@set.fintype_union _ (classical.dec_eq α) _ _ hs ht⟩
 
+lemma infinite_of_finite_compl {α : Type} [_root_.infinite α] {s : set α}
+  (hs : sᶜ.finite) : s.infinite :=
+λ h, set.infinite_univ (by simpa using hs.union h)
+
+lemma infinite_compl_of_finite {α : Type} [_root_.infinite α] {s : set α}
```

Comment:
```suggestion
lemma finite.infinite_compl {α : Type} [_root_.infinite α] {s : set α}
```
So that, if given `hs : s.finite`, one can say `hs.infinite_compl : sᶜ.infinite`.